lua/wikis/commons/Infobox/Map.lua

```diff
@@ -75,41 +83,72 @@ end
 
 --- Allows for overriding this functionality
 ---Add wikispecific categories
+---@protected
 ---@param args table
----@return table
+---@return string[]
 function Map:getWikiCategories(args)
 	return {}
 end
 
 --- Allows for overriding this functionality
 ---Adjust Lpdb data
+---@protected
 ---@param lpdbData table
 ---@param args table
 ---@return table
 function Map:addToLpdb(lpdbData, args)
 	return lpdbData
 end
 
+--- Allows for overriding this functionality
+---Returns the respective game for this map
+---@param args table
+---@return string?
+function Map:getGame(args)
+	return Game.name{game = args.game}
+end
```

Comment:
As there are no other overrides besides the error on AoE,
how about we just push the parsed game into self, and add the aoe-specific error in run or createInfobox?